tough/src/lib.rs

```diff
@@ -50,24 +50,31 @@ use crate::error::Result;
 use crate::fetch::{fetch_max_size, fetch_sha256};
 /// An HTTP transport that includes retries.
 #[cfg(feature = "http")]
-pub use crate::http::{HttpTransport, HttpTransportBuilder, RetryRead};
+pub use crate::http::{HttpTransport, HttpTransportBuilder};
+use crate::io::is_dir;
 use crate::schema::{
     DelegatedRole, Delegations, Role, RoleType, Root, Signed, Snapshot, Timestamp,
 };
 pub use crate::target_name::TargetName;
+use crate::transport::IntoVec;
 pub use crate::transport::{
     DefaultTransport, FilesystemTransport, Transport, TransportError, TransportErrorKind,
 };
 pub use crate::urlpath::SafeUrlPath;
+use async_recursion::async_recursion;
+pub use async_trait::async_trait;
+use bytes::Bytes;
```

Comment:
This is in the interface:

```suggestion
pub use bytes::Bytes;
```